src/control-plane/backend/lambda/api/service/reporting.ts

```diff
@@ -825,5 +825,64 @@ export class ReportingServ {
     return result;
   };
 
+  async warmup(req: any, res: any, next: any) {
+    try {
+      logger.info('start to warm up reporting service');
+      logger.info(`request: ${JSON.stringify(req.body)}`);
+
+      const projectId = req.body.projectId;
+      const appId = req.body.appId;
+      const region = req.body.region;
+      const dataApiRole = req.body.dataApiRole;
+      const dashboardCreateParameters = req.body.dashboardCreateParameters as DashboardCreateParameters;
+
+      if (cachedContents === undefined) {
+        cachedContents = new CachedContents();
+      }
+      const cachedObjs = await cachedContents.getCachedObjects(projectId, appId, region, dataApiRole);
+
+      //warm up redshift serverless
+      if (dashboardCreateParameters.redshift.newServerless) {
+        const input = {
+          Sqls: [`select * from ${appId}.ods_events limit 1`],
+          WorkgroupName: dashboardCreateParameters.redshift.newServerless?.workgroupName ?? undefined,
+          Database: projectId,
+          WithEvent: false,
+          ClusterIdentifier: dashboardCreateParameters.redshift.provisioned?.clusterIdentifier ?? undefined,
```

Comment:
You already know it's Redshift Serverless, `ClusterIdentifier` and `DbUser` are useless.

Reply:
fixed